### ISSUE TITLE
VoiceOver Improvements for search bar

### ISSUE THREAD

**Issue by caitbonnar** [OneBusAway/onebusaway-iphone#109]
Currently when selecting the search bar in VoiceOver, it is not apparent that you are searching by Route. When the user double taps to select the search bar, it should automatically say something like, "Button selected: Search by Route" and then the next thing selectable by swiping should be the button to search by Address then by Stop Number. 

Moved to https://github.com/OneBusAway/onebusaway-iphone/issues/127 :
Another thought on making the map view more accessible is to make the streets unable to be selected by VoiceOver. I'm looking into doing more extensive accessibility testing once the fall quarter starts to get more data on this, but from what I've heard so far, it may be more useful to have the stops on the map view selectable rather than the streets, where VoiceOver would read the stop name (crossroads) and number.

**Comment by aaronbrethorst** [comment]
Fixed with the search bar enhancements made earlier this year.
